src/test/java/com/google/step/servlets/PromoCodeServletTest.java

```diff
@@ -31,7 +31,10 @@
     private PromoCodeServlet servlet;
 
     private static final String NONEXISTENT_CHANNEL_ID = "NONEXISTENT_CHANNEL_ID";
+    private static final String COMPLETE_CHANNEL_ID = "https://www.youtube.com/channel/CHANNEL_ID";
```

Comment:
Instead of string use type URL go/tott-494 (In the google drive).  Also rename to CHANNEL_ID_URL